docs/technical_reference/unit_models/lt_med_surrogate.rst

```diff
@@ -0,0 +1,82 @@
+Low Temperature - Multi-effect Distillation (LT-MED)
+====================
+This Low Temperature Multi-effect Distillation (LT-MED) unit model
+   * supports steady-state only
+   * is a surrogate model
+   * is verified against the operation data in Plataforma Solar de Almeria (PSA)
+
+#TODO: Add index/reference to home page
+
+
+Degrees of Freedom
+------------------
+The LT-MED model has at least 5 degrees of freedom that should be fixed for the unit to be fully specified.
+
+Typically, the following variables are fixed, including the state variables at the inlet:
+    * Feed salinity (30-60 ":math:`\text{g/}\text{L}`" = ":math:`\text{kg/}\text{m}^3`")
+    * Feed temperature (15-35 ":math:`\text{C}`")
+    * Heating steam temperature (60-85 ":math:`\text{C}`")
+    * Recovery ratio (30%- 50%)
+    * System capacity (> 2000 ":math:`\text{m}^3\text{/day}`")
```

Comment:
Explicitly state the valid ranges being due to surrogate

Reply:
Message added.